torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -83,11 +83,21 @@ def _forward(self, input):
         running_std = torch.sqrt(self.bn.running_var + self.bn.eps)
         scale_factor = self.bn.weight / running_std
         scaled_weight = self.weight_fake_quant(self.weight * scale_factor.reshape([-1, 1, 1, 1]))
-        # this does not include the conv bias
-        conv = self._conv_forward(input, scaled_weight)
-        conv_orig = conv / scale_factor.reshape([1, -1, 1, 1])
-        if self.bias is not None:
-            conv_orig = conv_orig + self.bias.reshape([1, -1, 1, 1])
+
```

Comment:
A comment on the original code: I dont follow the logic here: We seem to be calling conv_forward , which includes the bias, though the comment says that "this does not include the conv bias"

Reply:
Our original implementation is complicated and can be simplified, we first add a bias as part of the call to _conv_forward, Then we again add the bias to conv_orig. All of this is removed by the batch norm op as part of subtracting the batch mean.

Reply:
> # this does not include the conv bias

yeah, this comment was not describing the code well

Reply:
A cleaner implementation is to do:
``` python
scaled_weight = self.weight_fake_quant(self.weight * scale_factor.reshape([-1, 1, 1, 1]))/scale_factor.reshape([-1,1,1,1])
F.convolution(input, scaled_weight, self.bias, self.stride, padding, self.dilation, self.groups)
```
```

Reply:
good point.  The code can potentially be made simpler with that refactor, thanks!

from what we discussed in chat, we'd still need a separate function for the above, because we want to insert the fake_quant on conv.weight directly at the op boundary of the aten op (to fit with how the rest of the code works and avoid custom logic).  So, since fake_quant is divided by scale_factor above, it wouldn't work out of the box.